build.gradle

```diff
@@ -45,7 +45,9 @@ allprojects {
       exceptionFormat = 'full'
     }
   }
+  tasks.withType(JavaExec) {
```

Comment:
What is this change needed for? I may have missed it, but I didn't see anything referencing the propagated properties.